src/PropertySuggester/Suggesters/SimpleSuggester.php

```diff
@@ -172,14 +172,15 @@ public function suggestByItem( Item $item, $limit, $minProbability, $context ) {
 
 				$entityId = $dataValue->getEntityId();
 
-				if ( !( $entityId instanceof Int32EntityId ) ) {
+				if ( !( $entityId instanceof ItemId ) ) {
 					throw new LogicException(
-						$entityId->getSerialization() . ' expected to be Int32EntityId'
+						"Property $numericPropertyId in wgPropertySuggesterClassifyingPropertyIds"
+						. ' does not have property type wikibase-item'
```

Comment:
we are still checking the value complies with the data type (and value) constraints